packages/core/saved-objects/core-saved-objects-server-internal/src/routes/import.ts

```diff
@@ -47,12 +47,17 @@ export const registerImportRoute = (
           {
             overwrite: schema.boolean({ defaultValue: false }),
             createNewCopies: schema.boolean({ defaultValue: false }),
+            compatibilityMode: schema.boolean({ defaultValue: false }),
           },
           {
             validate: (object) => {
               if (object.overwrite && object.createNewCopies) {
                 return 'cannot use [overwrite] with [createNewCopies]';
               }
+
+              if (object.createNewCopies && object.compatibilityMode) {
+                return 'cannot use [createNewCopies] with [compatibilityMode]';
+              }
```

Comment:
__note:__ If the API consumer explicitly requires new IDs then compatibility mode doesn't make sense.